project/accounts/factory.py

```diff
@@ -20,7 +20,7 @@ class Meta:
     first_name = factory.Faker("lorem")
     last_name = factory.Faker("lorem")
     about_me = factory.Faker("lorem")
-    is_verified = factory.fuzzy.FuzzyChoice(choices=[False, False, False, False])
+    is_verified = factory.fuzzy.FuzzyChoice(choices=[False, False])
```

Comment:
Should this also allow `True`?
```suggestion
    is_verified = factory.fuzzy.FuzzyChoice(choices=[True, False])
```

Reply:
I was not so sure. Because the field had False as a default value. But I will make the changes @brylie